src/assets/stylesheets/ui-root.scss

```diff
@@ -177,3 +178,109 @@
     margin: 0 12px;
   }
 }
+
+:local(.message-entry) {
+  position: relative;
+  margin: 8px 24px 24px 24px;
+  height: 48px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: white;
+  border: 1px solid #e2e2e2;
+  border-radius: 16px;
+
+  :local(.message-entry-input) {
+    @extend %default-font;
+    pointer-events: auto;
+    appearance: none;
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    outline-style: none;
+    background-color: transparent;
+    color: black;
+    padding: 8px 1.25em;
+    line-height: 2em;
+    font-size: 1.1em;
+    width: 100%;
+    border: 0px;
+    height: 32px;
+    margin-right: 100px;
+  }
+
+  :local(.message-entry-input)::placeholder{
+    color: $light-grey;
+    font-weight: 300;
+    font-style: italic;
+  }
+
+  :local(.message-entry-submit) {
+    @extend %action-button;
+    position: absolute;
+    right: 12px;
+    height: 32px;
+    min-width: 80px;
+  }
+}
+
+:local(.message-entry-in-room) {
+  @media(max-width: 900px) {
+    display:none;
+  }
+
+  position: absolute;
+  left: 16px;
+  bottom: 20px;
+  width: 33%;
+  height: 48px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: $darker-grey;
+  border-radius: 16px;
+  pointer-events: auto;
+  opacity: 0.3;
+  transition: opacity 0.25s linear;
+
+  :local(.message-entry-input-in-room) {
+    @extend %default-font;
+    appearance: none;
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    outline-style: none;
+    background-color: transparent;
+    color: white;
+    padding: 8px 1.25em;
+    line-height: 2em;
+    font-size: 1.1em;
+    width: 100%;
+    border: 0px;
+    height: 32px;
+    margin-right: 100px;
```

Comment:
This is almost identical to `.message-entry-input`. Maybe use a mixin or shared class?